modules/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BLangVM.java

```diff
@@ -415,14 +415,32 @@ private void exec() {
 
                 case InstructionCodes.LENGTHOF:
                     i = operands[0];
-                    j = operands[1];
-                    if (sf.refRegs[i] == null) {
-                        handleNullRefError();
+                    cpIndex = operands[1];
```

Comment:
Please refactor this instruction code to a new method.

Reply:
Refactored.